Cargo.toml

```diff
@@ -14,17 +14,17 @@ exclude = ["autogenerator/*"]
 serde_json = { version = '1.0.64', default-features = false, features = ["alloc"], optional = true }
 serde = { version = "1.0.136", default-features = false, features = ["derive", "alloc"], optional = true }
 hex = { version = "0.4", default-features = false , features = ["alloc"]}
-sodalite = { version = "0.4.0" }
+sodalite = { version = "0.4.0", default-features = false }
```

Comment:
Should we add `"sodalite/std"` to the `std` array in `[features]` below then?

Reply:
Well, it hasn't failed but you're probably right. Let's do it for consistency.

Reply:
Actually, it [seems](https://github.com/codyps/sodalite/blob/master/Cargo.toml) it doesn't have any `std` feature!

Reply:
Okay, nevermind then 😅

Reply:
Doesn't that mean that the `default-features = false` here does not have a purpose? The only feature in `sodalite` seems to be the optional `rand` dependency. But that is possibly exactly the dependency we want to avoid here (?)